test/end2end_test.go

```diff
@@ -1268,6 +1268,9 @@ func testUnknownHandler(t *testing.T, e env, unknownHandler grpc.StreamHandler)
 	}
 }
 
+// TestHealthCheckServingStatus fails with
+// "race detected during execution of test" when initialWindowSize and
```

Comment:
I don't think we should put this comment in this fork... Will be un-needed noise for merges and such. Consider filing a bug upstream.